### PR TITLE
Release/1.0.0

### PR DESCRIPTION
**Pull request by cs-ankit-gupta**

### CODE REVIEW

File: info.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+    "name": "connectorHealthMonitoring",
+    "version": "1.0.0",
+    "type": "solutionpack",
+    "local": true,
+    "label": "Connector Health Monitoring",
+    "dependencies": [],
+    "prerequisite": null,
+    "publisher": "Fortinet",
```
Comment (cs-rohit-khune) [inline info.json:9]: Please make solution pack as "certified" as true

File: info.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "description": "Solution Pack send the connector health status notification via email based on provided connector list.",
+    "help": null,
+    "category": [],
+    "supportInfo": null,
```
Comment (cs-rohit-khune) [inline info.json]: Please add "Fortinet Customer Support" as support info value

File: info.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    "iconLarge": null,
+    "fsrMinCompatibility": "7.2.0",
+    "date": "2022-08-23T12:29:30+00:00",
+    "contents": {
+        "playbooks": [
+            {
+                "name": "10 - SP - \tConnector Health Monitoring"
```
Comment (cs-rohit-khune) [inline info.json]: Please make a correction in the PB collection name to remove `\t`

File: info.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    "version": "1.0.0",
+    "type": "solutionpack",
+    "local": true,
+    "label": "Connector Health Monitoring",
+    "dependencies": [],
+    "prerequisite": null,
+    "publisher": "Fortinet",
+    "certified": "true",
+    "description": "Solution Pack send the connector health status notification via email based on provided connector list.",
+    "help": null,
```
Comment (cs-suyog-jinturkar) [inline info.json]: help link is missing

File: info.json [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    "name": "connectorHealthMonitoring",
+    "version": "1.0.0",
+    "type": "solutionpack",
+    "local": true,
+    "label": "Connector Health Monitoring",
+    "dependencies": [],
+    "prerequisite": null,
+    "publisher": "Fortinet",
+    "certified": "true",
+    "description": "Solution Pack send the connector health status notification via email based on provided connector list.",
```
Comment (cs-suyog-jinturkar) [inline info.json]: @cs-abhishek-shukla , please review this and suggest changes, if required

File: info.json [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+    "help": null,
+    "category": [],
```
Comment (cs-suyog-jinturkar) [inline info.json:13]: Category should be 'health monitoring' or so.. @cs-rohit-khune please provide your guidance here

**Review by cs-suyog-jinturkar — COMMENTED** [review]
Added few review comments.
Also, remove .DS_Store from everywhere

File: info.json (hunk 1 above)
Comment (cs-ankit-gupta) [reply info.json:9]: Done

File: info.json (hunk 2 above)
Comment (cs-ankit-gupta) [reply info.json]: Done

File: info.json (hunk 3 above)
Comment (cs-ankit-gupta) [reply info.json]: Done

File: info.json (hunk 5 above)
Comment (cs-abhishek-shukla) [reply info.json]: ```     "description": "Solution Pack sends connector health status notification via email based on user-specified connectors list.", ```

File: info.json (hunk 4 above)
Comment (cs-ankit-gupta) [reply info.json]: Done

File: info.json (hunk 6 above)
Comment (cs-ankit-gupta) [reply info.json:13]: @cs-suyog-jinturkar  Discussed with Rohit, we have two category as "Monitoring" and "System Monitoring". So which one we should pick for this SP?

File: info.json (hunk 5 above)
Comment (cs-ankit-gupta) [reply info.json]: Committed

File: info.json (hunk 6 above)
Comment (cs-suyog-jinturkar) [reply info.json:13]: Monitoring as this is not related to sysmon

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**